about.md

```diff
@@ -16,6 +16,15 @@ Można nas znaleźć na poniższych sieciach społecznościowych:
 
 [Pobierz statut w formacie PDF](https://github.com/riichi/statute-pl/releases/download/2021-06-17-1ee2a19/statut.pdf)
 
+### Jak dołączyć?
+
+Zapraszamy wszystkich chętnych do zostania członkami Stowarzyszenia!
+Zgodnie z §12 Statutu, w tym celu należy opłacić składkę, złożyć deklarację członkowską oraz uzyskać rekomendację dwóch członków Stowarzyszenia.
```

Comment:
```suggestion
Zgodnie z §12 Statutu, w tym celu należy opłacić składkę, uzyskać rekomendację dwóch członków Stowarzyszenia, oraz złożyć deklarację członkowską.
```
Złożenie deklaracji członkowskiej must happen after uzyskanie rekomendacji dwóch członków Stowarzyszenia, so maybe it makes sense to describe the process chronologically?

Reply:
Makes sense. Does Oxford comma function in Polish though?

Reply:
Sorry, indeed it doesn't.
```suggestion
Zgodnie z §12 Statutu, w tym celu należy opłacić składkę, uzyskać rekomendację dwóch członków Stowarzyszenia oraz złożyć deklarację członkowską.
```